runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -231,7 +231,7 @@ public class StreamingDataflowWorker {
   private final BoundedQueueExecutor workUnitExecutor;
   private final WindmillServerStub windmillServer;
   private final Thread dispatchThread;
-  private final Thread commitThread;
+  @VisibleForTesting final List<Thread> commitThreads;
```

Comment:
nit: how about ImmutableList and use builder below to create it.  Simpler to read code knowing it doesn't change size.

Reply:
done